Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.10-slim
+
+WORKDIR /app
+
+COPY app/ /app/
+
+RUN pip install requests
```

Comment:
It's recommended to use a `requirements.txt` file to manage Python dependencies. This allows for better version control and easier updates. You can copy the `requirements.txt` file into the image and run `RUN pip install -r requirements.txt` instead.